Vagrantfile

```diff
@@ -553,16 +557,13 @@ sudo chown -R vagrant:vagrant /vagrant
 
 rm -f /vagrant/provisioned_at
 rm -f /vagrant/version
-rm -f /vagrant/vvv-custom.yml
 
 touch /vagrant/provisioned_at
 echo `date "+%Y%m%d-%H%M%S"` > /vagrant/provisioned_at
 
 # copy over version and config files
 cp -f /home/vagrant/version /vagrant
-cp -f /home/vagrant/vvv-custom.yml /vagrant
 
-sudo chmod 0644 /vagrant/vvv-custom.yml
```

Comment:
Wondering if we put that to avoid some conflicts or other issues on provisions on some systems. what do you think @tomjn?

Reply:
@msaggiorato better to give back that command :-) and is ready to be tested